__tests__/articles.test.js

```diff
@@ -67,6 +69,8 @@ describe("/api/articles/:article_id", () => {
                 expect(typeof article).toBe("object");
                 expect(article).toHaveProperty('article_id');
                 expect(article.article_id).toBe(3);
+                expect(article).toHaveProperty('comment_count');
```

Comment:
we shouldn't have to change previous tests when new functionality has been added as that means the original test was brittle - something to note

Reply:
Cheers Heather, the challenge I ran into here was that the original test article ID didn't appear to have any comments to count. I'll just generate a new test next time.